server/tls.go

```diff
@@ -0,0 +1,81 @@
+package server
```

Comment:
https://github.com/giantswarm/microkit/pull/70/commits/a3d9e170f22fb1cf4b6309d8ddd23ca738e10dc0

Now it's in `tls` package.

I was thinking about prefixing microkit packages with `u` as it's close to `μ`. Then we would have `uerror`, `ulogger`, `utls`, etc. And we could avoid awkward imports.
WDYT?